Sources/XcodeProj/Objects/Sourcery/Equality.generated.swift

```diff
@@ -259,7 +259,7 @@ extension XCBuildConfiguration {
     /// :nodoc:
     func isEqual(to rhs: XCBuildConfiguration) -> Bool {
         if baseConfigurationReference != rhs.baseConfigurationReference { return false }
-        if !NSDictionary(dictionary: buildSettings).isEqual(NSDictionary(dictionary: rhs.buildSettings)) { return false }
+        if buildSettings != rhs.buildSettings { return false }
```

Comment:
I wasnt able to find info on how to run `Sourcery` to do generate this, but this makes tests pass.  Happy to make sure the generated code is correct but will need an assist on running things.

Reply:
I think it's ok. Let's make those changes manually.